db/migrate/20200406192204_create_users.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :email
+      t.integer :chat_id
```

Comment:
Maybe use `telegram_username`  or `telegram_id` instead of `chat_id`? It’s more explicit and future-proof in case we add support for other messaging services?